tests/system/HTTP/CURLRequestTest.php

```diff
@@ -1109,4 +1109,24 @@ public function testUserAgentOption(): void
         $this->assertArrayHasKey(CURLOPT_USERAGENT, $options);
         $this->assertSame($agent, $options[CURLOPT_USERAGENT]);
     }
+
+    public function testMultipleHTTP100(): void
+    {
+        $output = 'HTTP/1.1 100 Continue
+        Mark bundle as not supporting multiuse
+        HTTP/1.1 100 Continue
+        Mark bundle as not supporting multiuse
+        HTTP/1.1 200 OK
+        Server: Werkzeug/2.2.2 Python/3.7.17
+        Date: Sun, 28 Jan 2024 06:05:36 GMT
+        Content-Type: application/json
+        Content-Length: 33
+        Connection: close';
```

Comment:
The out is the following. This is not a valid HTTP response.
```
HTTP/1.1 100 Continue
        Mark bundle as not supporting multiuse
        HTTP/1.1 100 Continue
        Mark bundle as not supporting multiuse
        HTTP/1.1 200 OK
        Server: Werkzeug/2.2.2 Python/3.7.17
        Date: Sun, 28 Jan 2024 06:05:36 GMT
        Content-Type: application/json
        Content-Length: 33
```
And it is better to add comment showing the line break code is `CRLF`.

Reply:
Ok I'll handle this at this weekend.